### PR TITLE
The return of tintless shader

### PR DESCRIPTION
**Pull request by themuflon**

### CODE REVIEW

File: loom/graphics/gfxShader.h [hunk 1]
```diff
@@ -178,4 +180,18 @@ GLint uMVP;
     virtual void bind();
 };
 
+// Just like DefaultShader, but withot tinting.
```
Comment (bengarney) [inline loom/graphics/gfxShader.h]: withoUt

**Comment by bengarney** [comment]
One nit!

**Comment by bengarney** [comment]
Cool, will land when green.

**Comment by themuflon** [comment]
Looks like this was merged with JG-OptionalBlending

**Pull request merged**